R/normalize-terra.R

```diff
@@ -53,7 +53,7 @@ polygonData.SpatVector <- function(obj) {
 assure_crs_terra <- function(x) {
```

Comment:
Can we drop this function? 

https://github.com/rstudio/leaflet/pull/808#issuecomment-1533950559